staff/gianluca-bragaglia/pre-course/bingo-pro-html/bingo-pro-HTML.js

```diff
@@ -0,0 +1,329 @@
+// BINGO PRO
+
+
+var arrPlayers = [];
+var playerName;
+var status;
+
+function bingo() {
+
+    var playerPoints = 100;
+    var card = [];
+    var arrRandomNum = [];
+    var countA;
+    var countB;
+    var countC;
+    addNumberCard();
+    var card15 = [];
+    var card510 = [];
```

Comment:
maybe you can put all these variables in an object

Reply:
Maybe in the future, I will try